src/gossip/config/generate.lisp

```diff
@@ -1,16 +1,28 @@
 (in-package :gossip/config)
 
+(defparameter *nominal-gossip-port* 65002 "Duplicate of same symbol in gossip package
+    to avoid package interdependencies")
+
+#+OBSOLETE
```

Comment:
Why not? #+(or) means nothing to me; #+obsolete signals this code probably needs to be removed at some point but I'm leaving it here for reference until the new code is thoroughly tested.

Reply:
> The feature obsolete may be present, #+(or) will never evaluate to truth.

If the feature obsolete is present, it's only because somebody put it there. In which case I'd have to ask why.
I also use the feature #+IGNORE sometimes during development. Same answer.